stow/functions.php

```diff
@@ -52,47 +63,25 @@ function stow_setup() {
 				),
 			)
 		);
-
-		// Add child theme editor color pallete to match Sass-map variables in `_config-child-theme-deep.scss`.
-		add_theme_support(
-			'editor-color-palette',
-			array(
-				array(
-					'name'  => __( 'Primary', 'stow' ),
-					'slug'  => 'primary',
-					'color' => '#404040', 
-				),
-				array(
-					'name'  => __( 'Secondary', 'stow' ),
-					'slug'  => 'secondary',
-					'color' => '#f25f70', 
-				),
-				array(
-					'name'  => __( 'Dark Gray', 'stow' ),
-					'slug'  => 'foreground-dark',
-					'color' => '#111111',
-				),
-				array(
-					'name'  => __( 'Gray', 'stow' ),
-					'slug'  => 'foreground',
-					'color' => '#444444',
-				),
-				array(
-					'name'  => __( 'Light Gray', 'stow' ),
-					'slug'  => 'foreground-light',
-					'color' => '#767676',
-				),
-				array(
-					'name'  => __( 'White Smoke', 'stow' ),
-					'slug'  => 'background',
-					'color' => '#f0f0f0',
-				),
-			)
-		);
 	}
 endif;
 add_action( 'after_setup_theme', 'stow_setup', 12 );
 
+function stow_editor_color_palette( $palette ) {
+	$palette[] = array(
+		'name'  => __( 'Dark Gray', 'stow' ),
```

Comment:
Shouldn't this be dynamic and change with the palette? Looks like Stow's editor color palette were static, but some themes had them change when the user selects a new palette, like Alves does: https://github.com/Automattic/themes/blob/6862cbc6552a5fac9d9592c681588f461c32c3be/alves/functions.php#L94